src/components/TableComplex/RemoveDeviceDialog.react.js

```diff
@@ -19,14 +18,12 @@ class RemoveDeviceDialog extends Component {
 
   componentDidMount = () => {
     let field = document.getElementById('returnDiv');
+    let removeDeviceBtn = document.getElementById('removeDeviceButton');
     let fieldWidth = field.style.width;
     field.style.padding = '0px';
     field.style.color = '#900';
-    
-    let removeDeviceBtn = document.getElementById('removeDeviceButton');
     removeDeviceBtn.style.width = fieldWidth + 6;
     removeDeviceBtn.style.transition = 'none';
-    
     let deviceName = document.getElementById('devicename').parentNode;
     deviceName.style.width = fieldWidth - 16;
```

Comment:
dont change styling like this, use the reactjs way.

Reply:
I have made use of react refs for the same purpose (grabbing the DOM elements)